aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -422,17 +422,17 @@ def main_aggregator(files_to_agg, var_to_agg, site_code, input_dir='', output_di
 if __name__ == "__main__":
 
     parser = argparse.ArgumentParser(description="Aggregate ONE variable from ALL instruments from ALL deployments from ONE site")
-    parser.add_argument('-site', dest='site_code', help='site code, like NRMMAI',  required=True)
-    parser.add_argument('-var', dest='varname', help='variable to aggregate, like TEMP', required=True)
-    parser.add_argument('-files', dest='filenames',
+    parser.add_argument('--site', dest='site_code', help='site code, like NRMMAI',  required=True)
+    parser.add_argument('--var', dest='varname', help='variable to aggregate, like TEMP', required=True)
+    parser.add_argument('--files', dest='filenames',
```

Comment:
While we're at it, we could simplify both the code and the command line options by:
1) Turning `required` args into positional ones (so don't need `required=True`);
1) Making the argument names consistent with the destination names used within the code; and
1) Then remove the unnecessary `dest` argument (set from the long option by default - see [docs](https://docs.python.org/3.5/library/argparse.html#dest)).

So e.g. this would become
```suggestion
    parser.add_argument('site_code', help='site code, like NRMMAI')
    parser.add_argument('varname', help='variable to aggregate, like TEMP')
    parser.add_argument('filenames',
                        help='name of the file that contains the source URLs (relative to inpath, if given)')
```